pkg/services/contract/contract.go

```diff
@@ -110,8 +110,8 @@ var ErrMissingPublicURL = errors.New("missing publicUrl")
 
 func (s *service) configureContracts() (err error) {
 	if s.config.ActingPartyCn == "" {
-		err = ErrMissingActingParty
-		return
+		// todo remove this check in 0.17
+		logrus.Info("no actingPartyCn configured, this is needed for v2 contracts (deprecated)")
 	}
```

Comment:
shouldn't we also add a notice that actingPartyCN is going to be removed in the next version?

Reply:
docs are still todo

Reply:
When 0.16 is done, we must encourage everyone to use v3 contracts.